### PR TITLE
[test] Fix ninja test for `join` @open sesame 12/03 12:35

### PR DESCRIPTION
**Pull request by anyj0527**
- Currently `$ ninja test` failed for `join`
- Add gst_plugin_path and ext_plugin_path for env

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -506,14 +506,15 @@ if get_option('enable-test')
   )
 
   path_gst_plugin = join_paths(meson.build_root(), 'gst', 'nnstreamer')
+  path_gst_join_plugin = join_paths(meson.build_root(), 'gst', 'join')
```
Comment (dongju-chae) [inline meson.build:509]: I recommend the below style.
```
diff --git a/meson.build b/meson.build
index 8ee8623..10e7ced 100644
--- a/meson.build
+++ b/meson.build
@@ -505,7 +505,9 @@ if get_option('enable-test')
     configuration: nnstreamer_test_conf
   )
 
-  path_gst_plugin = join_paths(meson.build_root(), 'gst', 'nnstreamer')
+  path_gst_plugin = join_paths(meson.build_root(), 'gst')
+  path_ext_plugin = join_paths(meson.build_root(), 'ext')
+
   path_nns_conf = join_paths(meson.build_root(), 'nnstreamer-test.ini')
   path_nns_plugin_prefix = join_paths(meson.build_root(), 'ext', 'nnstreamer')
   path_nns_plugin_filters = join_paths(path_nns_plugin_prefix, 'tensor_filter')
@@ -513,7 +515,7 @@ if get_option('enable-test')
   path_nns_plugin_converters = join_paths(path_nns_plugin_prefix, 'tensor_converter')
 
   testenv = environment()
-  testenv.set('GST_PLUGIN_PATH', path_gst_plugin + ':' + path_nns_plugin_prefix)
+  testenv.set('GST_PLUGIN_PATH', path_gst_plugin + ':' + path_ext_plugin)
   testenv.set('NNSTREAMER_CONF', path_nns_conf)
   testenv.set('NNSTREAMER_FILTERS', path_nns_plugin_filters)
   testenv.set('NNSTREAMER_DECODERS', path_nns_plugin_decoders)
```

You may refer to `debian/rules` and `packaging/nnstreamer.spec` to set `GST_PLUGIN_PATH`.

Comment (gichan-jang) [inline meson.build:509]: How about changing like this? 
I modified other files as below. I think I missed this part. 
(i.e., line 662 at nnstreamer.spec)
```suggestion
  path_gst_plugin = join_paths(meson.build_root(), 'gst')
```

Comment (jaeyun-jung) [reply meson.build]: I agree DJ's suggestion

Comment (anyj0527) [reply meson.build]: I updated the code to follow your kind suggestion. THANKS!

File: meson.build [hunk 2]
```diff
@@ -505,15 +505,17 @@ if get_option('enable-test')
     configuration: nnstreamer_test_conf
   )
 
-  path_gst_plugin = join_paths(meson.build_root(), 'gst', 'nnstreamer')
+  path_gst_plugin = join_paths(meson.build_root(), 'gst')
+  path_ext_plugin = join_paths(meson.build_root(), 'ext')
+
   path_nns_conf = join_paths(meson.build_root(), 'nnstreamer-test.ini')
   path_nns_plugin_prefix = join_paths(meson.build_root(), 'ext', 'nnstreamer')
```
Comment (gichan-jang) [inline meson.build:512]: Minor revision request.
```suggestion
  path_nns_plugin_prefix = join_paths(path_ext_plugin, 'nnstreamer')
```

Comment (anyj0527) [reply meson.build]: Thanks! this is more neat. I updated the code

**Review by dongju-chae — APPROVED** [review]
LGTM~

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**